packages/@ember/-internals/glimmer/lib/component-managers/custom.ts

```diff
@@ -144,6 +145,14 @@ export interface ComponentArguments {
   named: Dict<unknown>;
 }
 
+export function initialRenderInstrumentDetails(component: any): any {
+  return { object: `<${component.constructor.name}:${guidFor(component)}:gc>`, initialRender: true };
```

Comment:
The existing implementation in the `curly` component manager uses `component.instrumentDetails` to generate the `object` string, which in turn relies on the `toString` function of `EmberObject`. I wasn't really sure the best way forward here so just used this for now.